research-software-directory/RSD-as-a-service
# On error show no content message

# On error show no content message

Fixes #63 

How to test:

* start only frontend without backend and visit the software page. You should see the website and simple no content message in the middle of the page.

![image](https://user-images.githubusercontent.com/9204081/146761401-9b79b103-8f7a-49ad-8218-525617aabdfa.png)
